partner_name_vat/models/models.py

```diff
@@ -17,3 +17,7 @@ def name_get(self):
     @api.depends("vat")
     def _compute_display_name(self):
         super()._compute_display_name()
+        for partner in self:
+            if partner.vat:
+                partner.display_name = f"{partner.display_name} ({partner.vat})"
+        return
```

Comment:
Is necessary to implement the code that was already implemented by Odoo via "show_vat" in the previous version? Does "show_vat" still exists on v17.0?
Also if name_get does not longer exist on v17.0 you can remove it as well.